e2e-tests/functions

```diff
@@ -5,29 +5,61 @@ ROOT_REPO=${ROOT_REPO:-$(realpath ../../..)}
 source "${ROOT_REPO}/e2e-tests/vars.sh"
 test_name=$(basename "$(pwd)")
 
-deploy_operator() {
-	kubectl -n "${NAMESPACE}" apply --server-side --force-conflicts -f "${DEPLOY_DIR}/crd.yaml"
-	kubectl -n "${NAMESPACE}" apply --server-side --force-conflicts -f "${DEPLOY_DIR}/rbac.yaml"
-
-	yq eval \
-		"$(printf 'select(documentIndex==0).spec.template.spec.containers[0].image="%s"' "${IMAGE}")" \
-		"${DEPLOY_DIR}/operator.yaml" \
-		>"${TEMP_DIR}/operator.yaml"
+create_namespace() {
+	local namespace=$1
+	if [[ ${CLEAN_NAMESPACE} == 1 ]]; then
+		kubectl delete namespace $namespace --ignore-not-found || :
+		kubectl wait --for=delete namespace "$namespace" || :
+	fi
 
-	case ${test_name} in
-		telemetry-transfer)
-			yq eval \
-				"$(printf 'select(documentIndex==0).spec.template.spec.containers[0].env[4]={"name":"PERCONA_VS_FALLBACK_URI","value":"%s"}' "http://version-service:11000")" \
-				"${TEMP_DIR}/operator.yaml" \
-				| yq eval 'select(documentIndex==0).metadata.labels["helm.sh/chart"]="fake-operator-chart"' - \
-					>"${TEMP_DIR}/vs.operator.yaml"
-
-			mv "${TEMP_DIR}/vs.operator.yaml" "${TEMP_DIR}/operator.yaml"
-			;;
-		*) ;;
-	esac
+	kubectl create namespace $namespace
+}
 
-	kubectl -n "${NAMESPACE}" apply -f ${TEMP_DIR}/operator.yaml
+deploy_operator() {
+	if [ -n "$OPERATOR_NS" ]; then
```

Comment:
Can't we just have `if` for cw rbac and operator and have just one case for `telemetry-transfer` case?
Or even just have a special case in the telemetry test for this and not in deploy_operator?

Reply:
I want to discuss it. To move it in telemetry test looks good for me. I will have a look. 
We should have if anyway since we have different amount of env params in cw and not cw cases.